changelogs/fragments/461-ec2_instance-wait_sanity.yml

```diff
@@ -0,0 +1,9 @@
+breaking_changes:
+    - ec2_instance - if plays require the old behavior of ``ec2_instance``, waiting for
+      EC2 instance monitoring status to become ``OK`` when launching a new instance,
+      the action will need to specify ``state: started`` (https://github.com/ansible-collections/community.aws/pull/461).
+bugfixes:
+    - ec2_instance - ``ec2_instance`` was waiting on EC2 instance monitoring status
+      to be ``OK`` when launching a new instance. This could cause a play to wait
+      multiple minutes for AWS's monitoring to complete status checks
+      (https://github.com/ansible-collections/community.aws/pull/461).
```

Comment:
```suggestion
breaking_changes:
    - | 
      ec2_instance - if plays require the old behavior of ``ec2_instance``, waiting for
      EC2 instance monitoring status to become ``OK`` when launching a new instance,
      the action will need to specify ``state: started`` (https://github.com/ansible-collections/community.aws/pull/461).
bugfixes:
    - | 
      ec2_instance - ``ec2_instance`` was waiting on EC2 instance monitoring status
      to be ``OK`` when launching a new instance. This could cause a play to wait
      multiple minutes for AWS's monitoring to complete status checks
      (https://github.com/ansible-collections/community.aws/pull/461).
```